docs/docs/pixabay-source-plugin-tutorial.md

```diff
@@ -356,7 +356,11 @@ Your plugin is ready. Restart `npm run develop` and open a browser at [http://lo
 }
 ```
 
-Or [open the query from this link](<http://localhost:8000/___graphql?query=%7B%0A%20%20allPixabayPhoto(limit%3A%2010)%20%7B%0A%20%20%20%20edges%20%7B%0A%20%20%20%20%20%20node%20%7B%0A%20%20%20%20%20%20%20%20largeImageURL%0A%20%20%20%20%20%20%20%20pageURL%0A%20%20%20%20%20%20%20%20tags%0A%20%20%20%20%20%20%20%20user%0A%20%20%20%20%20%20%7D%0A%20%20%20%20%7D%0A%20%20%7D%0A%7D%0A>).
+Or open the query from this link:
+
+```text
+http://localhost:8000/___graphql?query=%7B%0A%20%20allPixabayPhoto(limit%3A%2010)%20%7B%0A%20%20%20%20edges%20%7B%0A%20%20%20%20%20%20node%20%7B%0A%20%20%20%20%20%20%20%20largeImageURL%0A%20%20%20%20%20%20%20%20pageURL%0A%20%20%20%20%20%20%20%20tags%0A%20%20%20%20%20%20%20%20user%0A%20%20%20%20%20%20%7D%0A%20%20%20%20%7D%0A%20%20%7D%0A%7D%0A
+```
```

Comment:
i'am not shure how to deal with this link? 
_note: it is just an convinience link with the graphql some lines above._

Reply:
I would treat this as an exception to the rule in the style guide since the text explicitly informs the reader that they may click on the link but I'll defer to @gatsbyjs/learning

Reply:
Ya, I'm ok with that since it's such a gnarly one to see in text format.

Reply:
reverted the change